devel/simple-websocket-server/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.1
+
+name	  	    simple-websocket-server
+version		    2.0.0-rc4
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen @repagh openmaintainer
```

Comment:
Same as for the other port. Use `{` and `}` around your email and GitHub handle to associate them.

Reply:
done